src/component/common/ProtectedRoute/ProtectedRoute.jsx

```diff
@@ -11,7 +11,7 @@ const ProtectedRoute = ({
             {...rest}
             render={props => {
                 if (unauthorized) {
-                    return <Redirect to="/login" />;
+                    return <Redirect to={'/login'} />;
```

Comment:
I guess this makes it relative?

Reply:
No, but it works because of the basename set in the router: 

```
 <Router basename={`${getBasePath()}`}>
   ... children
</Router>
```

Due to this property our internal routing will be automatically handled by react-router-dom (prepending the basename as the subpath) as long as we use their components to set up and link to routes, leaving the API paths for us to handle.